generate_mk_docs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+poetry run mkdocs build
```

Comment:
Could you add a step here to open `config/website_config.json` print the selected value to the cli and also verify it is one of the acceptable values (and error if not - saying what the acceptable values are).  This would save us making a typo and not realising it until we navigate to the site.

Reply:
done

Reply:
jq doesn't seem to be installed by default. Can we add this to the project using poetry so it is in the toml file

Reply:
In the error message can you list the allowed types please

Reply:
And print a line saying "Building content for _type_"